inscriptions/ISic001408.xml

```diff
@@ -246,9 +246,9 @@
                         <citedRange>32</citedRange>
                         <ptr target="http://zotero.org/groups/382445/items/MH7A3CFV"/>
                     </bibl>
-                    <bibl>
-                        <author>Jeffery</author>
-                        <date>1990</date>
```

Comment:
Because Valentina and I decided to have two different references for Jeffrey, one for the first edition and one for the second. For the second I'm adding Johnston as a second author. Have replaves all instances of this globally, so this is very likely one of the conflicts between this batch and my more recent bibliography edits.